packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -110,29 +119,29 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
     valueToInt
   )
 
-  /**
```

Comment:
Was removing this documentation block intentional?

Reply:
It was not, nice catch